pizza_app/css/style.css

```diff
@@ -0,0 +1,251 @@
+/* basic rules */
+
+* {
+    box-sizing: border-box;
+}
+
+html {
+    background: #252839;
+    color: #ebebd3;
+    font-family: 'Roboto', sans-serif;
+    font-size: 14px;
+}
+
+/* HEADER */
+
+.page-header {
+    position: fixed;
+    top: 0;
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 10px 0;
+    color: #f2b632;
+    background: #1f2230;
+    border-bottom: 2px solid #f2b632;
+}
+
+.page-header .current-time {
+    margin-left: 10px;
+}
+
+.page-header .current-time .fa-clock {
+    display: none;
+}
+
+.page-header .title {
+    display: flex;
+    align-items: center;
+}
+
+.page-header .title img {
+    width: 40px;
+    height: 40px;
+}
+
+.page-header .title h1 {
+    font-family: 'Lobster', sans-serif;
+    font-size: 24px;
+}
+
+/* MAIN */
+
+.page-main {
+    margin: 60px 20px 20px 20px;
+    padding: 10px 0;
+}
+
+/* FOOTER */
+
+.page-footer {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: flex-start;
+    padding: 10px;
+    background: #1f2230;
+    border-top: 2px solid #f2b632;
+}
+
+.page-footer .address {
+    font-style: normal;
+}
+
+.page-footer .address a {
+    color: #f2b632;
+}
+
+.page-footer .copyright {
+    padding-top: 10px;
+    font-size: 12px;
+}
+
+/* BUTTONS */
+
+.btn {
+    color: #1f2230;
+    background: #f2b632;
+    border: 1px solid transparent;
+    border-radius: 5px;
+    font-size: 14px;
+    font-weight: bold; 
+    text-transform: uppercase;
+    box-shadow: -1px 1px 8px rgba(0, 0, 0, 0.6);
+    cursor: pointer;
+}
+
+.btn:hover,
+.btn:focus {
+    background: #f5cb5c;
+    box-shadow: -2px 2px 16px rgba(0, 0, 0, 0.8);
+    outline: none;
+}
+
+.btn:active {
+    background: #e8dcb9;
+    box-shadow: -4px 4px 24px rgba(0, 0, 0, 0.8);
+    outline: none;
+}
+
+.btn-wrapper {
+    display: flex;
+}
+
+.page-header .btn {
+    width: 40px;
+    height: 30px;
+}
+
+.page-header .btn-add img {
+    width: 20px;
+}
+
+.page-header .btn-login {
```

Comment:
This kind of selector does not look clear. It can be any button and in some moment you may get mess and fear of breaking something on changes.

You may have something like following:
```html
<button class="btn btn-primary btn-login">Login</button>
```

However, I am not an expert in your case because I did not read code deep.

Reply:
@viktor-yakubiv I have only one login button, so it's clear for me